### PR TITLE
feature: Added hover information support

### PR DESCRIPTION
**Pull request by JordyMackay**

### CODE REVIEW

File: server/src/test/position.test.ts [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+                availableObjects,
+                link,
+                callbackfn
+            );
+
+            // ASSERT
+            expect(result.length).to.equal(0);
+        });
+
+        it("should return one objects when defined", () => {
```
Comment (emielb) [inline server/src/test/position.test.ts:194]: not very important, but still
```suggestion
        it("should return one object when defined", () => {
```

File: server/src/test/position.test.ts [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+                callbackfn
+            );
+
+            // ASSERT
+            expect(result.length).to.equal(1);
+            expect(result[0].linkUrl).to.equal(linkUrlString);
+            expect(result[0].omtLocalObject).to.deep.equal(availableObjects.definedObjects[0])
+        });
+
+        it("should return one objects when imported and defined in import", () => {
```
Comment (emielb) [inline server/src/test/position.test.ts:216]: same here
```suggestion
        it("should return one object when imported and defined in import", () => {
```

File: server/src/omtAvailableObjectsProvider.ts [hunk 3]
```diff
@@ -107,16 +116,30 @@ function findModelEntries(modelEntries: Record<string, unknown>, documentText: s
...
+        localDefinedObjects.push({ name: key, range, parameters });
     });
     return localDefinedObjects;
 }
 
+interface ObjectWithParams {
+    params: string[];
+}
+
+function instanceWithParams(object: unknown | ObjectWithParams): object is ObjectWithParams {
```
[the remark this answers is not in the document]
Comment (JordyMackay) [reply server/src/omtAvailableObjectsProvider.ts]: I think that will work as well, I will change it to `object: unknown`

File: server/src/omtAvailableObjectsProvider.ts [hunk 4]
```diff
@@ -60,16 +60,25 @@ export function getAvailableObjectsFromDocument(document: TextDocument, shorthan
  */
 function findDefinedObjects(value: string, documentText: string, define: string): OmtLocalObject[] {
     const localDefinedObjects: OmtLocalObject[] = [];
-    const queriesRegex = new RegExp(`(?<=DEFINE ${define} )([a-zA-Z0-9]+)`, "gm");
+    const queriesRegex = new RegExp(`(?<=DEFINE ${define} )([a-zA-Z0-9_]+)`, "gm");
```
[the remark this answers is not in the document]
Comment (JordyMackay) [reply server/src/omtAvailableObjectsProvider.ts]: I added the dash here, and updated the unittest.

**Review by emielb — APPROVED** [review]

**Pull request merged**